yt/visualization/volume_rendering/scene.py

```diff
@@ -409,6 +410,11 @@ def save_annotated(
             If True, will render the scene before saving.
             If False, will use results of previous render if it exists.
             Default: True
+        tf_rect : seqeunce, optional
+           A rectangle that defines the location of the transfer
+           function legend for the case where there are multiple
+           volume sources with associated transfer functions.  tf_rect
```

Comment:
so you're saying this argument has no effect if there's only one volume source w/ a transfer function ?
I'm thinking we could log a warning if the argument is passed but not used. Additionally I think the doctoring should make it crystal clear wether the argument is consumed or not.

Reply:
I've clarified this